app/components/bookings/create/create-booking.component.ts

```diff
@@ -1,4 +1,5 @@
 import {Component} from '@angular/core';
+import {TranslatePipe} from "ng2-translate/ng2-translate";
```

Comment:
@mamahler: well, the point of merging is, to merge stuff and not to overwrite it ;)
This will most likely throw a merge conflict, or be auto-merged and just silently be added to the final `Component`, which should be avoided